man/docker-run.1.md

```diff
@@ -327,11 +327,14 @@ reservation. So you should always set the value below **--memory**, otherwise th
 hard limit will take precedence. By default, memory reservation will be the same
 as memory limit.
 
-**--memory-swap**=""
-   Total memory limit (memory + swap)
+**--memory-swap**="LIMIT"
+   A limit value equal to memory plus swap. Must be used with the  **-m**
+(**--memory**) flag. This value should always larger than **-m**(**--memory**)
+value.
 
-   Set `-1` to disable swap (format: <number>[<unit>], where unit = b, k, m or g).
-This value should always larger than **-m**, so you should always use this with **-m**.
+   The format of `LIMIT` is <`number`>[<`unit`>]. Unit can be `b` (bytes),
```

Comment:
hm, I just noticed @moxiegirl provided this in her example; @moxiegirl did you do so with a specific reason? It differs from the other places where we show a "format"